gallery/visualtests/nxFormSelect.js

```diff
@@ -16,7 +16,7 @@ describe('nx-form-select', function() {
   });
 
   describe('Simple NxFormSelect', function() {
-    it('has a light border when pristine', simpleTest(selector));
+    it('has a dark border when pristine', simpleTest(selector));
```

Comment:
The distinction between this test and the one that follows is no longer necessary.  I'd have just one, implemented using `simpleTest`, named `it('has a dark border by default'`.  And then maybe rename the hover test to say "darker" border.

Reply:
> The distinction between this test and the one that follows is no longer necessary. I'd have just one, implemented using `simpleTest`, named `it('has a dark border by default'`. And then maybe rename the hover test to say "darker" border.

Thanks, updated: 867bfa96276d741db73a1dc17d89e8ba58d04042